ui/apps/everest/vite.config.ts

```diff
@@ -11,6 +11,7 @@ export default defineConfig({
       ignored: path.resolve(__dirname, '.e2e/**/*.*'),
     },
     proxy: {
+      '/v1/policies': 'http://127.0.0.1:8000',
```

Comment:
what's running on port 8000?

Reply:
I forgot this from when I was mocking the policies API. Thanks for the notice!